### PR TITLE
Quirks for Smartthings MultiV4

### PR DESCRIPTION
**Pull request by roblandry**

### CODE REVIEW

**Comment by roblandry** [comment]
I am assuming this one will be left open until the accelerometer cluster can be properly implemented, but sending this one also due to battery reports working.

File: zhaquirks/smartthings/multiv4.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    PollControl, Ota, BinaryInput
+from zigpy.zcl.clusters.security import IasZone
+from zhaquirks.centralite import PowerConfigurationCluster
+from zigpy.zcl.clusters.measurement import TemperatureMeasurement
+from zigpy.quirks import CustomDevice
+
+
+class SmartThingsMultiV4(CustomDevice):
+
+    def __init__(self, *args, **kwargs):
```
Comment (dmulcahey) [inline zhaquirks/smartthings/multiv4.py]: unneeded can remove the entire function

File: zhaquirks/smartthings/multiv4.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        super().__init__(*args, **kwargs)
+
+    signature = {
+        # <SimpleDescriptor endpoint=1 profile=260 device_type=1026
+        # device_version=0
+        # input_clusters=[0, 1, 3, 15, 32, 1026, 1280, 64514]
+        # output_clusters=[25]>
+        1: {
```
Comment (dmulcahey) [inline zhaquirks/smartthings/multiv4.py]: we match on man / model now so add them here

File: zhaquirks/smartthings/__init__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import zigpy.types as t
```
Comment (dmulcahey) [inline zhaquirks/smartthings/__init__.py]: drop this file from the PR and import this from zigpy quirks

Comment (dmulcahey) [reply zhaquirks/smartthings/__init__.py]: scratch that... move it to Centralite and we have to change it too... https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/b2c942bd2220e7f5367465c49b8460f9edf70763/devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy#L236

File: zhaquirks/smartthings/multiv4.py (hunk 1 above)
Comment (roblandry) [reply zhaquirks/smartthings/multiv4.py]: Done

File: zhaquirks/smartthings/multiv4.py (hunk 2 above)
Comment (roblandry) [reply zhaquirks/smartthings/multiv4.py]: Done

**Pull request merged**